### ISSUE TITLE
Minor UX concept: make the "or create new insight" part of the modal after you click the big inviting "+"

### ISSUE THREAD

**Issue by Joelkw** [sourcegraph/sourcegraph#24486]
When a user first clicks to "add" insights, they realize they can't actually use the "+" to create. This is inconsistent with where we use the "+" on the creation buttons on the top right. Also, as a user, I keep hitting the "+" and then having to exit the modal to actually hit the creation path. I would guess I am perhaps not unique.

**Comment by vovakulikov** [comment]
@Joelkw I agree. But note that we already have a link to the creation UI. Probably we should change or somehow emphasize Hey here is the creation UI. 

<img width="324" alt="Screenshot 2021-09-06 at 21 15 16" src="https://user-images.githubusercontent.com/18492575/132251796-cd3242f5-c97d-4db8-b148-1a71c512ff30.png">


Also a kind of unrelated thing but I think we should move selected insights on top of the insight list, cause at the moment when you have a lot of insight it's complicated to find some insight and uncheck it. Event if we have a good search bar. cc @AlicjaSuska

**Comment by Joelkw** [comment]
@vovakulikov good point (re already having it) – I missed that! Probably good datapoint that it's not prominent enough. 

Ah, yeah, I think that's also a good concept – issue is here now #24712

**Comment by unclejustin** [comment]
I think the simplest solution would be redesigning this card with the two options. Create or Add. @AlicjaSuska thoughts?

![image](https://user-images.githubusercontent.com/1855233/134043121-f28c59fd-51bd-427c-9258-959d3abf9140.png)

**Comment by vovakulikov** [comment]
@unclejustin @AlicjaSuska If we want to change this card we also should consider re-visit this card for the built-in dashboard as well (personal, org-level, global) Cause at these dashboards we have a different version of this card. (exactly opposite logic rather than on the screenshot above from Justin) - it is just create insight button cause at these dashboards you can't add other insights like this - there is one way and this is create insight with particular visibility (personal, org, global) to see that insight on some built-in dashboard personal, org-level, global)
